dev/NavigationView/NavigationView_rs1_themeresources.xaml

```diff
@@ -1095,16 +1092,18 @@
                                 <VisualState x:Name="IconOnly">
                                     <VisualState.Setters>
                                         <Setter Target="PointerRectangle.Visibility" Value="Visible" />
-                                        <Setter Target="LayoutRoot.Width" Value="48" />
+                                        <Setter Target="LayoutRoot.MinWidth" Value="48" />
                                         <Setter Target="ContentPresenter.Visibility" Value="Collapsed" />
                                         <Setter Target="SelectionIndicatorGrid.Margin" Value="4,0,4,4" />
```

Comment:
>4,0,4,4 [](start = 94, length = 7)

I see what you mean about the margin resource now.. I think this could make sense..

Reply:
Yep, I am tracking those in https://github.com/microsoft/microsoft-ui-xaml/issues/2877 but I can also add the Margin specific resources as part of this PR (while I'm at it).